controlpanel/api/models/policys3bucket.py

```diff
@@ -21,5 +18,16 @@ class Meta:
         unique_together = ("policy", "s3bucket")
         ordering = ("id",)
 
+    @property
     def aws_name(self):
         return self.policy.arn
+
+    def grant_bucket_access(self):
+        cluster.Group(self.policy).grant_bucket_access(
```

Comment:
What's a group? I must have missed something in the last few iterations.

Reply:
> Represents a group of cluster users who have access to datasources

Reply:
@s-block created the `IAMManagedPolicy` model and the `PolicyS3Bucket` which is essentially a group of users having access to an S3Bucket. We decided not to call the class `Group`, because it might be confusing with the `django.contrib.auth.models.Group`, but in retrospect, that was probably a mistake, and `IAMManagedPolicy` is a terrible name.

Reply:
I'm planning to fix that, but I wanted to avoid making any DB migrations in this PR.

Reply:
(but probably we can beef up the class comment to make it more clear why it's done, etc...)

Reply:
oh hang on, I got confused, I thought the DB model was called `Group` but it's actually called `IAMManagedPolicy` so maybe we can rename the `cluster` class or maybe we stick with `cluster.IAMManagedPolicy` for consistency for the time being 🤷‍♂